bootcd/genesis.ks.template

```diff
@@ -220,6 +220,8 @@ echo ">>>>> Removing bloat from /usr/share"
 rm -rf /usr/share/doc
 rm -rf /usr/share/icons
 
+sed -e 's|ACTIVE_CONSOLES.*|ACTIVE_CONSOLES="/dev/tty1 /dev/ttyS0 /dev/ttyS1"|' -i /etc/sysconfig/init
```

Comment:
As a note of clarification I've been testing with the `Dockerfile` not `Dockerfile.centos7` which may have different defaults.